lib/browse_everything.rb

```diff
@@ -4,8 +4,6 @@
 require 'browse_everything/retriever'
 
 module BrowseEverything
-  class InitializationError < RuntimeError; end
```

Comment:
Does this exception class need to be moved to the helper?  It seems like it might break apps if the exception class is renamed.

Reply:
This has been moved to the `BrowseEverything` Module:
https://github.com/samvera/browse-everything/pull/189/commits/7ac541e34ccd9a7b91811e503f3ec1253d351372#diff-d9f2a3b3a4c2e755ea28e3e468addbbdR23